kani-compiler/src/kani_middle/transform/check_aliasing/function_cache.rs

```diff
@@ -0,0 +1,66 @@
+pub use stable_mir::ty::GenericArgKind as GenericArg;
```

Comment:
Renaming of commonly used structures makes it harder to track what's going on. I'd rather avoid that.